cmd/argocd/commands/project.go

```diff
@@ -363,8 +417,15 @@ func NewProjectRemoveOrphanedIgnoreCommand(clientOpts *argocdclient.ClientOption
 		name string
 	)
 	var command = &cobra.Command{
-		Use:   "remove-orphaned-ignore PROJECT GROUP KIND NAME",
+		Use:   "remove-orphaned-ignore PROJECT GROUP KIND",
```

Comment:
This is the place where I would like to be reassured that I didn't make a bad decision: in fact I understood that you can use `NAME` only in combination with the `--name` flag as in `argocd proj add-oprhaned-ignore`, so I removed it from usage and put the example with `--name NAME`. Is my understanding correct?

Reply:
Yep, based on this I agree

```go
			if len(args) != 3 {
				c.HelpFunc()(c, args)
				os.Exit(1)
			}
```

Great catch!